config.toml.example

```diff
@@ -0,0 +1,4 @@
+[Node-Details]
+node_name = "Example"
+node_admin_email = "admin@example.com"
+login_home = "index.html"
```

Comment:
comment this to show that this is the default behaviour (add more comments if you think that it's unclear)

Reply:
Commented entries in "config.toml.example"

Reply:
No, not all of them. Only "login_home" should be commented because that is the only one with a default value.

Reply:
Uncommented the lines with no default value.